docs/units-and-global-variables.rst

```diff
@@ -77,6 +77,7 @@ Block and Transaction Properties
 - ``blobhash(uint index) returns (bytes32)``: versioned hash of the ``index``-th blob associated with the current transaction.
   A versioned hash consists of a single byte representing the version (currently ``0x01``), followed by the last 31 bytes
   of the SHA256 hash of the KZG commitment (`EIP-4844 <https://eips.ethereum.org/EIPS/eip-4844>`_).
+  Returns zero if no blob with the given index exists.
```

Comment:
We should probably include that info in the opcode table as well: https://docs.soliditylang.org/en/develop/yul.html#evm-dialect

The builtin also listed in the cheatsheet: https://docs.soliditylang.org/en/develop/cheatsheet.html#block-and-transaction-properties. Not sure if we should add it there or not though. Cheatsheet is meant to be concise. On the other hand the description is the same as here so it would make sense to have it in both places as well.